src/js/controller/settings/exportimage/GifExportController.js

```diff
@@ -177,7 +177,7 @@
         link : imageUrl,
         shortLink : this.shorten_(imageUrl, URL_MAX_LENGTH, '...')
       });
-      this.uploadStatusContainerEl.innerHTML = `${this.i18n.gifExportSettingSectionYourImageAvailableAt()} ${linkHtml}`;
+      this.uploadStatusContainerEl.innerHTML = this.i18n.gifExportSettingSectionYourImageAvailableAt() + ' :' + linkHtml;
```

Comment:
I think this is from another PR, right?

Reply:
I applied to all the other branches since template literals was not working, I will get errors in the terminal saying that it could not identify the character ` so instead I did string concatenation